agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryChannelTest.java

```diff
@@ -78,11 +80,13 @@ private TelemetryChannel getTelemetryChannel() throws MalformedURLException {
             .policies(policies.toArray(new HttpPipelinePolicy[0]))
             .httpClient(recordingHttpClient);
     LocalFileCache localFileCache = new LocalFileCache(tempFolder);
+    StatsbeatModule mockedStatsModule = Mockito.mock(StatsbeatModule.class);
     return new TelemetryChannel(
         pipelineBuilder.build(),
         new URL(END_POINT_URL),
         new LocalFileWriter(localFileCache, tempFolder, null),
-        null);
+        mockedStatsModule,
+        true);
```

Comment:
this looks like a test of the normal (non-statsbeat) channel
```suggestion
        false);
```

Reply:
yes, i made it to statsbeat channel so that i don't need to mock more stuff (e.g. mockstatsModule.getNetworkStatsbeat will return null).  kind of cheating but you caught me :)